README.md

```diff
@@ -1,3 +1,52 @@
 # ClusterFuzzLite
+ClusterFuzzLite is a continuous fuzzing solution that runs as part of
```

Comment:
Do you want a link to a definition on "fuzzing?" It's fine if it was a choice to leave it out.